### PR TITLE
M2C and CV List of Druids

### PR DESCRIPTION
**Pull request by SaravShah**
Able to run a list of druids from a CSV file through m2c and cv. 

- The file assumes a CSV file w/ no header and no druid prefix.

- update readme
- create rake task
- create Checksum and MoabToCatalog methods
- RSpec tests

connects #821

### CODE REVIEW

File: lib/audit/moab_to_catalog.rb [hunk 1]
```diff
@@ -20,6 +21,12 @@ def self.check_existence_for_druid(druid)
     logger.info "#{Time.now.utc.iso8601} M2C check_existence_for_druid ended for #{druid}"
   end
 
+  def self.check_existence_for_druid_list(druid_list_file_path)
+    CSV.foreach(druid_list_file_path) do |row|
+      MoabToCatalog.check_existence_for_druid(row.first)
```
Comment (ndushay) [inline lib/audit/moab_to_catalog.rb]: output the errors?

File: lib/audit/checksum.rb [hunk 2]
```diff
@@ -57,4 +58,11 @@ def self.validate_druid(druid)
   ensure
     logger.info "#{Time.now.utc.iso8601} CV validate_druid ended for #{druid}"
   end
+
+  # assumes that the list of druids is in column 1, and has no header.
+  def self.validate_list_of_druids(druid_list_file_path)
+    CSV.foreach(druid_list_file_path) do |row|
+      Checksum.validate_druid(row.first)
```
Comment (ndushay) [inline lib/audit/checksum.rb]: output the errors?

**Comment by SaravShah** [comment]
@ndushay ready for review!

File: lib/audit/moab_to_catalog.rb (hunk 1 above)
Comment (atz) [reply lib/audit/moab_to_catalog.rb]: How is this CSV input?  Seems like a single column of text.

Comment (SaravShah) [reply lib/audit/moab_to_catalog.rb]: I could use`File.open` then use `.each_line` on that file but this seemed easier?

Comment (atz) [reply lib/audit/moab_to_catalog.rb]: If that's what he starts from, I guess... 🤷‍♂️ Would be nice to enable taking the values from STDIN, but whatever works.

**Pull request merged**